chain/jsonrpc/src/lib.rs

```diff
@@ -312,6 +314,9 @@ impl JsonRpcHandler {
         Ok(match request.method.as_ref() {
             // Handlers ordered alphabetically
             "block" => process_method_call(request, |params| self.block(params)).await,
+            "broadcast_tx" => {
```

Comment:
Usually, we add new methods with the `EXPERIMENTAL_` prefix, is there any reason you don't follow this tradition?

Reply:
Yes. I don't think `EXPERIMENTAL_` prefix helps us, tbh. It just gives the methods like `EXPERIMENTAL_tx_status` which are de-facto standards and nobody cares. It's like `1.0` version in Rust
I still think it's a good idea to add this prefix for new/debug/experimental things, but here I only pack 2 old methods to a more generic one, it doesn't sound experimental to me.

Reply:
> It's like never using 1.0 version in Rust

That's a good analogy and I agree. :+1:

Reply:
> why we can't break the backwards compat

Because nobody knows if these methods are used in any of the community tools.

Reply:
[I'm sharing my personal opinion here, it may differ from the opinion of other employees]
The reason is that the reality does not always align with the doc. 
We can go and drop `EXPERIMENTAL_tx_status`, the doc clearly says that all experimental methods are unstable.
But it has existed for so long, and many users rely on it. If I drop it, people will complain, and, imho, it's not important if they have a "legal right" on these complaints. All we can do here is to try to minimize the negative experience.
So, yes, breaking the compatibility is a complex question, and there are no solid rules here. Each time we have to consider all the factors while making the decision.